tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{27,34,35,36}, flake8
+skip_missing_interpreters = true
+
+[testenv]
+deps = pytest
+       python-dotenv
+commands = py.test {posargs}
+passenv = VCAP_SERVICES
+
+[testenv:flake8]
+deps = flake8
```

Comment:
This is a bit weird, since `flake8` should pull `six` inside the virtualenv created by `tox`. Maybe we have different versions of tox that create slightly different virtualenvs.
Do the tests work if you just add `six` to the `[testenv:flake8]` section?